app/enterprise/changelog.md

```diff
@@ -4,6 +4,167 @@ no_search: true
+- The [Request Size Limiting](https://docs.konghq.com/hub/kong-inc/request-size-limiting/)
+(`request-size-limiting`) plugin has a new configuration `require_content_length` that
+causes the plugin to ensure a valid Content-Length header exists before reading the request body.
+- The [Serverless Functions](https://docs.konghq.com/hub/kong-inc/serverless-functions/)
+(`serverless-functions`) plugin introduces a sandboxing capability, is *enabled* by default,
+and where only Kong PDK, OpenResty `ngx` APIs, and Lua standard libraries are allowed.
+- The [LDAP Authentication Advanced](https://docs.konghq.com/hub/kong-inc/ldap-auth-advanced/) (`ldap-auth-advanced`) plugin has two new features:
+  - added config `log_search_results` that allows displaying all of the LDAP search results received from the LDAP server.
+  - additional debug log statements added for authenticated groups.
+- jq plugin
+- Rate Limiting Advanced:
+  - add a jitter (random delay) for the Retry-After header
+- mtls-auth
+  - add support for tags in the DAO
+  - plugin ensures the existence of any provided CAs when creating the plugin entry
+- Response transformer advanced:
+  - json paths for nested elements and arrays:
+  - transform gzipped content:
+- Collector
+  - Support for hybrid mode
+  - remove log_bodies configuration option
+- OpenID Connect Library
+  - Token introspection now checks the status code properly
+  - More consistent response body checks on HTTP requests  
+
+#### Configuration
+- `client_max_body_size` and `client_body_buffer_size`, that previously
+  hardcoded to 10m, are now configurable through
+  [nginx_admin_client_max_body_size](/gateway-oss/2.3.x/configuration/#nginx_http_client_max_body_size)
+  and [nginx_admin_client_body_buffer_size](/gateway-oss/2.3.x/configuration/#nginx_admin_client_body_buffer_size).
+- Kong-generated SSL private keys now have `600` file system permission.
+- Properties `ssl_cert`, `ssl_cert_key`, `admin_ssl_cert`,
+  `admin_ssl_cert_key`, `status_ssl_cert`, and `status_ssl_cert_key`
+  is now an array: previously, only an RSA certificate was generated
+  by default; with this change, an ECDSA is also generated. On
+  intermediate and modern cipher suites, the ECDSA certificate is set
+  as the default fallback certificate; on old cipher suite, the RSA
+  certificate remains as the default. On custom certificates, the first
+  certificate specified in the array is used.
+- {{site.base_gateway}} now runs as a `kong` user if it exists; if user does not exist
+  in the system, the `nobody` user is used, as before.
+
+### Dependencies
+- Bumped `kong-plugin-serverless-functions` from 1.0 to 2.1.
+- Bumped `lua-resty-dns-client` from 5.1.0 to 5.2.0.
+- Bumped `lua-resty-healthcheck` from 1.3.0 to 1.4.0.
+- Bumped `OpenSSL` from 1.1.1h to 1.1.1i.
+- Bumped `kong-plugin-zipkin` from 1.1 to 1.2.
+- Bumped `kong-plugin-request-transformer` from 1.2 to 1.3.
+
+### Fixes
+
+#### Core
```

Comment:
I had added a number of comments on these fixes in the last review and it looks like they still apply (e.g. https://github.com/Kong/docs.konghq.com/pull/2542#discussion_r565478826, https://github.com/Kong/docs.konghq.com/pull/2542#discussion_r565479499, https://github.com/Kong/docs.konghq.com/pull/2542#discussion_r565480083, https://github.com/Kong/docs.konghq.com/pull/2542#discussion_r565480408).